Makefile

```diff
@@ -39,3 +44,18 @@ format: format_jsonnet
 .PHONY: format_jsonnet
 format_jsonnet: $(JSONNET_FILES)
 	$(JSONNET_DOCKER) $(JSONNETFMT_ARGS) -- $?
+
+compile:
+	jb install
```

Comment:
`jb` is part of commodores container image. I suggest to use the following as `JB_CMD`.

```
docker run --rm --user="$(shell id -u)" --volume "${PWD}:/app/$(COMPONENT_NAME)" --workdir /app/$(COMPONENT_NAME) --entrypoint /usr/local/bin/jb projectsyn/commodore:latest install
```

Might be worth to factor out the common `DOCKER_OPTS`.